### PR TITLE
Logging fixes

### PR DESCRIPTION
**Pull request by dvoet**
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [ ] **Submitter**: Make sure documentation for code is complete
- [ ] **Submitter**: Make sure liquibase is updated if appropriate
- [ ] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [ ] Tell  that the PR exists if he wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [ ]  sign off
- [x] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Squash commits, rebase if necessary
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Merge to develop 
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -51,9 +51,9 @@ libraryDependencies ++= {
     ("com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev53-1.20.0"),
     "com.typesafe.slick" %% "slick" % slickV,
     "com.typesafe.slick" %% "slick-hikaricp" % slickV,
-    "org.slf4j" % "slf4j-nop" % "1.6.4",
```
Comment (dvoet) [inline build.sbt:54]: this was killing our logging

File: build.sbt [hunk 2]
```diff
@@ -51,9 +51,9 @@ libraryDependencies ++= {
...
     "mysql" % "mysql-connector-java" % "5.1.38",
-    "org.liquibase" % "liquibase-core" % "3.3.5"
+    "org.liquibase" % "liquibase-core" % "3.3.5",
+    "ch.qos.logback" % "logback-classic" % "1.1.6"
```
Comment (dvoet) [inline build.sbt:56]: this gives us a logging system other than nop

File: docker/run.sh [hunk 3]
```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-exec java -jar $(find /rawls -name 'rawls*.jar')
+exec java $JAVA_OPTS -jar $(find /rawls -name 'rawls*.jar')
```
Comment (dvoet) [inline docker/run.sh:5]: unrelated to logging but required for connecting to mysql

**Comment by helgridly** [comment]
seems legit :+1:

File: src/test/resources/logback-test.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@
         </encoder>
     </appender>
 
-    <root level="DEBUG">
+    <root level="INFO">
```
Comment (dvoet) [inline src/test/resources/logback-test.xml:18]: vastly reduces size of test logs and speed of tests

**Pull request merged**